Update to latest Azure CLI and Bicep.

edburns opened the pull request:
https://dev.azure.com/was-azure/was-azure/_workitems/edit/1158

edburns committed:
Update to latest Azure CLI and Bicep.

https://dev.azure.com/was-azure/was-azure/_workitems/edit/1158

majguo commented on external-deps-versions.properties:
```diff
@@ -2,9 +2,9 @@
+AZ_CLI_VERSION=2.74.0
 # Release version from https://github.com/Azure/bicep/releases
-BICEP_VERSION=v0.30.23
+BICEP_VERSION=v0.36.1
```
Suggest keeping `BICEP_VERSION` version unchanged as the twas vm image workflow failure seems not related it. Also, it aligns with Bicep version used in partner center, it shouldn't be updated unless the Bicep version used in partner center is updated.

majguo commented on external-deps-versions.properties:
```diff
@@ -2,9 +2,9 @@
 
 # Release version from https://github.com/Azure/azure-cli/releases
-AZ_CLI_VERSION=2.40.0
+AZ_CLI_VERSION=2.74.0
```
Besides the twas vm image workflows are referencing `AZ_CLI_VERSION` defined in this config file, pls make sure all other workflows in vendoers' repos (twas-single, twas-cluster, liberty-on-aks, liberty-on-aro, wls-on-azure and jboss-eap-on-azure) are still working with the version update here.

edburns committed:
On branch edburns/ibm-1158-ihs-image
modified:   external-deps-versions.properties

@majguo wrote:

> Suggest keeping BICEP_VERSION version unchanged as the twas vm image workflow failure seems not related it. Also, it aligns with Bicep version used in partner center, it shouldn't be updated unless the Bicep version used in partner center is updated.

Signed-off-by: Ed Burns <edburns@microsoft.com>
